### PR TITLE
[15.0][FIX] web_pivot_computed_measure: Take care about -Infinity value too

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT48536

ping @carlosdauden @sergio-teruel

### CODE REVIEW

File: web_pivot_computed_measure/static/src/pivot/pivot_renderer.esm.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ import {patch} from "web.utils";
 
 patch(PivotRenderer.prototype, "web_pivot_computed_measure.PivotRenderer", {
     getFormattedValue(cell) {
-        if (cell.value === Infinity) {
+        if (cell.value === Infinity || cell.value === -Infinity) {
```
Comment (chienandalu) [inline web_pivot_computed_measure/static/src/pivot/pivot_renderer.esm.js]: What about this?
```suggestion
        if (Math.abs(cell.value) === Infinity) {
```

**Comment by CarlosRoca13** [comment]
Changed, thanks :+1:

**Review by chienandalu — APPROVED** [review]

**Review by sergio-teruel — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge patch

**Comment by CarlosRoca13** [comment]
ping @pedrobaeza

**Pull request merged**